enterprise/api/v1/views.py

```diff
@@ -1024,7 +1024,9 @@ def get(self, request):
         """
         url = settings.TABLEAU_URL + '/trusted'
         enterprise_customer_uuid = get_enterprise_customer_from_user_id(request.user.id)
-        payload = {'username': enterprise_customer_uuid}
+        # Enterprise customer uuid is being store without hyphens in tableau
+        tableau_username = enterprise_customer_uuid.replace('-', '')
```

Comment:
str() required here?

Reply:
No, As `get_enterprise_customer_from_user_id ` function returning it as str.